ddtrace/contrib/aiohttp/patch.py

```diff
@@ -1,40 +1,375 @@
+import asyncio
+import functools
+import logging
+import sys
+
 from ddtrace import config
 from ddtrace.vendor import wrapt
 
-from ...pin import Pin
 from ...utils.wrappers import unwrap
+from ...propagation.http import HTTPPropagator
+from ...pin import Pin
+from ...ext import http as ext_http
+from ..httplib.patch import should_skip_request
 
+import aiohttp
+from yarl import URL
```

Comment:
I'm guessing `yarl` is pulled in by aiohttp so this works, but just wanted to make sure we don't need to add another dependency.

Reply:
yep